integration_tests/integration/book-recall.spec.js

```diff
@@ -269,6 +276,26 @@ context('Book a recall', () => {
     })
   })
 
+  it('User sees an error if invalid Last Release Prison is entered', () => {
```

Comment:
There's already a test for this - https://github.com/ministryofjustice/manage-recalls-ui/blob/main/integration_tests/integration/book-recall.spec.js#L282-L289

Reply:
Removing.